client/src/Pages/Infrastructure/Create/index.jsx

```
@@ -105,8 +188,34 @@
+
+		const { error } = infrastructureMonitorValidation.validate(
+			{ [name]: adjustedValue },
+			{ abortEarly: false }
+		);
+		setErrors((prev) => ({
+			...prev,
+			...(error ? { [name]: error.details[0].message } : { [name]: undefined }),
+		}));
+	};
+
+	const handleCheckboxChange = (event) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'handleCheckboxChange'. Consider to delete it or rename it.

'handleCheckboxChange' is defined here:


(lint/suspicious/noRedeclare)